### PR TITLE
Arachne 1649

### PR DESCRIPTION
**Pull request by MikMironov**
add check atlas connection button in DN

### CODE REVIEW

File: src/modules/Admin/components/SystemSettings/container.jsx [hunk 1]
```diff
@@ -81,6 +81,10 @@ class SystemSettingsBuilder extends ContainerBuilder {
       },
       applySettings: () => {
         dispatchProps.applySettings(() => dispatchProps.loadSystemSettings());
+      },
+      checkAtlas: () => {
+        dispatchProps.checkAtlas()
+          .catch((error) => {alert(error.errors._error)})
```
Comment (johnSamilin) [inline src/modules/Admin/components/SystemSettings/container.jsx]: I'd also added condition to mapStateToProps to show loader at the time of request

File: src/modules/Admin/const.js [hunk 2]
```diff
@@ -56,6 +56,7 @@ const apiPaths = {
 
   restartServer: () => '/api/v1/admin/restart',
   ping: () => '/api/v1/auth/me',
+  atlasConnection: () => '/api/v1/admin/atlasConnection',
```
Comment (pavgra) [inline src/modules/Admin/const.js]: atlas-connection

File: src/modules/Admin/components/SystemSettings/container.jsx [hunk 3]
```diff
@@ -81,6 +81,10 @@ class SystemSettingsBuilder extends ContainerBuilder {
       },
       applySettings: () => {
         dispatchProps.applySettings(() => dispatchProps.loadSystemSettings());
+      },
+      checkAtlas: () => {
+        dispatchProps.checkAtlas()
```
Comment (pavgra) [inline src/modules/Admin/components/SystemSettings/container.jsx]: do not see it in dispatch block

Comment (pavgra) [inline src/modules/Admin/components/SystemSettings/container.jsx]: we've talked about displaying atlas version on success

**Pull request merged**